src/network/send.cpp

```diff
@@ -3506,11 +3509,12 @@ PacketMessageUNICODE::PacketMessageUNICODE(const CClient* target, const nword* p
  ***************************************************************************/
 PacketDeath::PacketDeath(CChar* dead, CItemCorpse* corpse, bool fFrontFall) : PacketSend(XCMD_CharDeath, 13, PRI_NORMAL)
 {
+	UNREFERENCED_PARAMETER(fFrontFall);

```

Comment:
What about sending fFrontFall if the client version supports it, instead of sending 0 to everyone?

Reply:
I'm investigating which client mincliver supports it, but to make Sphere usable to everyone I did this modification for now.

Reply:
Okay then, merging!